### PR TITLE
Fixes broken search when absolute href is generated from location header

### PR DESCRIPTION
**Pull request by franckstauffer**
The enclosing test case shows that expandPath is breaking the search part of the relative URL it is trying to expand.

### CODE REVIEW

File: test/plugin-spec.js [hunk 1]
```diff
@@ -965,6 +967,46 @@ describe('Halacious Plugin', function () {
         });
     });
 
+    it.only('use of location header for absolute link generation should not break url search', function (done) {
```
Comment (bleupen) [inline test/plugin-spec.js]: remove .only

**Comment by bleupen** [comment]
looks good, thanks for the PR!

if you drop the .only from your test case i'll merge it and get a release out

**Comment by franckstauffer** [comment]
Done! Sorry for having left that 'only' !

**Comment by bleupen** [comment]
no prob thanks again!

**Pull request merged**